src/server/bundle.ts

```diff
@@ -12,7 +12,7 @@ let esbuildInitialized: boolean | Promise<void> = false;
 async function ensureEsbuildInitialized() {
   if (esbuildInitialized === false) {
     if (Deno.run === undefined) {
-      const wasmURL = new URL("./esbuild_v0.14.51.wasm", import.meta.url).href;
+      const wasmURL = new URL("./esbuild_v0.17.2.wasm", import.meta.url).href;
```

Comment:
I think we need to vendor this wasm at `src/server/esbuild_v0.17.2.wasm` (`src/server/esbuild_v0.14.51.wasm` is removed with this change)

Reply:
You're right... nice catch! We probably want CI / testing on this. Opening an issue to be addressed in another PR.